### DISCUSSION TITLE
Extension for Yandex browser

### DISCUSSION THREAD

**Discussion by Krovyakov-Ivan** [keepassxreboot/keepassxc#10814] [Ideas]
Good afternoon. Is it possible to add an extension for Yandex browser for Linux?

**Comment by varjolintu** [comment]
The browser is based on Chromium, so it should be enough to set it as a Custom Browser in KeePassXC's Browser Integration settings (Advanced tab). It needs the user's configuration path that includes the `native-messaging-hosts` in it. The placeholder works as an example. This is probably something like `.config/yandex-browser/native-messaging-hosts/`.
